components/apimgt/org.wso2.carbon.apimgt.rest.api.admin/src/main/java/org/wso2/carbon/apimgt/rest/api/admin/impl/ExportApiServiceImpl.java

```diff
@@ -90,27 +90,24 @@ public Response exportApplicationsGet(String appName, String appOwner, Boolean w
                 return Response.status(Response.Status.FORBIDDEN).entity(errorMsg).build();
             }
 
+            // clear all duplicate keys with tokens
+            applicationDetails.getKeys().clear();
+
```

Comment:
Check for the possibilities of NPE if applicationDetails.getKeys() returns null

Reply:
Keys are initialized in the `Application` class.